src/chapters/unit-testing/junit.rst

```diff
@@ -0,0 +1,172 @@
+JUnit
+=====
+
+JUnit is a library that provides the classes, methods, and assertions
+for writing and executing unit tests in Java. In this course, 
+we use JUnit4. Although there are newer versions now available, 
+JUnit4 is still popular and widely used.
+
+Java Annotations
+^^^^^^^^^^^^^^^^
+
+.. index:: ! annotations
+
+On the topic of unit testing, the annotation ``@Test`` is used to 
+indicate that a public void method should be treated as a test case.
+
+In Java, **annotations** are formalized bits of information about a program. They operate
+somewhere between actual code syntax and a comment on the code. Annotations do not 
+directly affect the code they annotate, but they can supply information to the compiler.
+Annotations are indicated with an ``@`` symbol. We will provide an example in the 
+walkthrough below.
+
+
+Testing Walkthrough
+^^^^^^^^^^^^^^^^^^^
+
+In your ``java-web-dev-exercises`` repo, we've included two ``.jar`` files for the JUnit 
+library. These are located in the ``lib`` folder. In the future, as you build your own
+Java projects, you will likely include these dependencies in a different fashion. Many Java
+projects use a build automation tool to help manage dependencies, but since this project 
+doesn't use one of these tools, we've included the ``.jar`` files.
+
+``main/Car`` and ``test/CarTest``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Within ``org.launchcode.java.demos``, open the ``testing`` directory. Inside ``testing``, you'll
+see a ``main`` directory and and ``test`` directory. Open the ``Car`` class within ``main`` and 
+look around. Here, we provide a class ``Car`` with basic information about a make, model, 
```

Comment:
Inside the ``Car`` class, there is a ternary conditional operator:
``double milesAbleToTravel = miles > maxDistance ? maxDistance : miles;``

This was never discussed in unit 1, so the syntax needs to be explained or replaced with an ``if/else`` statement.  Searching Google would be tricky, since there is no keyword to guide the query.